src/psyclone/transformations.py

```diff
@@ -4097,27 +3810,34 @@ def validate(self, node, options=None):
         '''
         Check that the supplied node is a valid target for this transformation.
 
-        :param node: the loop node to validate.
+        :param node: the PSyIR node to validate.
         :type node: :py:class:`psyclone.psyGen.CodedKern`
         :param options: a dictionary with options for transformations.
         :type options: dictionary of string:values or None
 
-        :raises TransformationError: if the supplied node is not a CodedKern
+        :raises TransformationError: if the supplied node is not a CodedKern.
```

Comment:
Please also document the exception raised if we're not using the NEMO API.

Reply:
Done. (with Gocean API you mean)

Reply:
Oops, yes and thanks.